canairio/settings/base.py

```diff
@@ -124,43 +110,17 @@
 
 TRACK_COLLECTION_NAME = 'test_tracks_data'
 
-FB_API_KEY = os.getenv('FB_API_KEY', '')
-FB_AUTH_DOMAIN = os.getenv('FB_AUTH_DOMAIN', '')
-FB_DATABASE_URL = os.getenv('FB_DATABASE_URL', '')
-FB_STORAGE_BUCKET = os.getenv('FB_STORAGE_BUCKET', '')
+# Firebase
+FB_API_KEY = ''
+FB_AUTH_DOMAIN = ''
+FB_DATABASE_URL = ''
+FB_STORAGE_BUCKET = ''
 
-INFLUXDB_HOST = os.getenv('INFLUXDB_HOST', '')
+INFLUXDB_HOST = getenvvar('INFLUXDB_HOST')
 INFLUXDB_PORT = 8086
 INFLUXDB_USERNAME = None
 INFLUXDB_PASSWORD = None
 INFLUXDB_DATABASE = 'canairio'
 INFLUXDB_TIMEOUT = 10
 
-ENVIRONMENT = os.getenv('ENVIRONMENT', ENVIRONMENT)
-if 'production' in ENVIRONMENT:
```

Comment:
The reason this exists is to be able to extend this to `if 'heroku' in os.environ:` or similar. This would allow us to create and deploy environments without ever having to modify code.

Reply:
Sounds good, I would like to learn about that.